src/Psalm/Internal/CallMap_74_delta.php

```diff
@@ -21,5 +21,6 @@
 'old' => [
     'password_hash' => ['string|false', 'password'=>'string', 'algo'=>'int', 'options='=>'array'],
     'password_needs_rehash' => ['bool', 'hash'=>'string', 'algo'=>'int', 'options='=>'array'],
+    'ReflectionProperty::getType' => ['?ReflectionType'],
```

Comment:
I think it should be in the `new` section, as it was added, not removed.

Reply:
Also you need to add this to `CallMap.php` itself. Deltas are used to patch the callmap for older versions.

Reply:
Adjusted both here and in `CallMap.php` :+1: